charts/dave/values.yaml

```
@@ -387,3 +414,8 @@ keycloak:
     extraEnvVars:
       - name: IMPORT_FILES_LOCATIONS
         value: /config/*
+    resources:
+      requests:
+        cpu: 100m
+        memory: 256Mi
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)